examples/forwarding-refs/customized-display-name.js

```diff
@@ -7,8 +7,8 @@ function logProps(Component) {
     return <LogProps {...props} forwardedRef={ref} />;
   }
 
-  // Give this component a more helpful display name in DevTools.
-  // e.g. "ForwardRef(logProps(MyComponent))"
+  // 在 DevTools 中为该组件提供一个更有用的显示名。
+  // 例如 “ForwardRef(logProps(MyComponent))”
```

Comment:
这是代码注释，不是正文，所以感觉不改也行，下文的一样，统一就好。
```suggestion
  // 例如“ForwardRef(logProps(MyComponent))”
```